examples/run-all.js

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env node
 
+/* eslint-disable */
```

Comment:
Eslint was unhappy with all the require statements and more in this file. Since this file isn't production code I defaulted to just disabling eslint altogether. Here are the errors I see for this file as an FYI (no require, __dirname is undefined, and no console):
<img width="676" alt="Screen Shot 2022-11-02 at 1 21 54 PM" src="https://user-images.githubusercontent.com/459878/199594950-00e77e0d-9b39-4ba6-b4bf-b7b515e8cada.png">